app/tests/spec/views/sign_in_bounced.js

```diff
@@ -0,0 +1,69 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+define((require, exports, module) => {
+  'use strict';
+
+  const { assert } = require('chai');
+  const { Model } = require('backbone');
+  const sinon = require('sinon');
+  const View = require('views/sign_in_bounced');
+  const template = require('stache!templates/sign_in_bounced');
+
+  describe('views/sign_in_bounced', () => {
+    let clickHandler, formPrefill, model, user, view;
+
+    beforeEach(() => {
+      formPrefill = {
+        clear: sinon.spy()
+      };
+      model = new Model();
+      model.set('email', 'foo@example.com');
+      user = {
+        removeAllAccounts: sinon.spy()
+      };
+      view = new View({ formPrefill, model, user });
+      clickHandler = view.events['click #create-account'];
+    });
+
+    it('set properties correctly', () => {
+      assert.isFunction(clickHandler);
+      assert.equal(view.template, template);
```

Comment:
I think you want a strictEqual here, I believe that equal will attempt type conversion.